packages/svelte/tests/snapshot/samples/inline-module-vars/_expected/client/index.svelte.js

```diff
@@ -0,0 +1,19 @@
+import "svelte/internal/disclose-version";
+import * as $ from "svelte/internal/client";
+
+const __DECLARED_ASSET_0__ = "__VITE_ASSET__2AM7_y_a__ 1440w, __VITE_ASSET__2AM7_y_b__ 960w";
```

Comment:
are these later inlined completely? it should be possible to have one static string here instead of interpolation

Reply:
I agree, but that's a bit more complicated to handle here and we've generally tried to avoid adding such complication here in favor of letting general purpose optimizers handle it. Unfortunately esbuild does not yet handle this case, but I've filed a feature request for it: https://github.com/evanw/esbuild/issues/3570. Oxc also does not handle it, so I've file a request there as well so that it will be handled when switching to rolldow, but it was unfortunately closed: https://github.com/oxc-project/oxc/issues/2646

Reply:
so in lieu of external tools supporting it, do we reconsider inlining these ourselves? it would make svelte output more svelte after all.

Reply:
potentially as a future enhancement if folks agree. i'm still hoping esbuild and oxc will add support. they haven't said they're against it, but just haven't prioritized it. I'd rather not try to do it as part of this PR though to keep things simple for now as this solves 95% of the issue